tests/testthat/test-infrastructure.r

```diff
@@ -59,3 +59,96 @@ test_that("use_data", {
     expect_equal(local_test_data_item_to_save, 42L)
   })
 })
+
+test_that("add_desc_package only modifies 'field' section of DESCRIPTION", {
```

Comment:
@hadley not sure if you're going to like this, but I broke up the tests and made their descriptions a bit more verbose... Let me know...